fhd_core/setup_metadata/fhd_struct_init_layout.pro

```diff
@@ -0,0 +1,268 @@
+function fhd_struct_init_layout, ant_table_header, ant_table_data, $
+    array_center = array_center, coordinate_frame = coordinate_frame, $
+    gst0 = gst0, earth_degpd = earth_degpd, ref_date = ref_date, time_system = time_system, $
+    dut1 = dut1, diff_utc = diff_utc, nleap_sec = nleap_sec, $
+    pol_type = pol_type, n_pol_cal_params = n_pol_cal_params, n_antenna = n_antenna, $
+    antenna_names = antenna_names, antenna_numbers = antenna_numbers, antenna_coords = antenna_coords, $
+    mount_type = mount_type, axis_offset = axis_offset, pola = pola, pola_orientation = pola_orientation, $
+    pola_cal_params = pola_cal_params, polb = polb, polb_orientation = polb_orientation, polb_cal_params = polb_cal_params
```

Comment:
I will also ensure that if they are passed in they aren't overridden by the info from the uvfits file.